Documentation/ColumnsConfig/Type/User/Index.rst

```diff
@@ -161,8 +161,10 @@ implementing a rendering. See :ref:`FormEngine docs
          <t3core:security-xss>`.
 
 
-   The array :php:`$this->data` provides the following data:
-
+   Array :php:`$this->data`: ...
+   
+   ::
+   
       * The row of the currently edited record in
```

Comment:
... and just reduce the indentation here.

Reply:
Shouldn't the block remain?

Reply:
Actually no, a plain list is enough. (Technically a table would be better but that's too much change at once.)